extensions/amp-subscriptions/0.1/local-subscription-platform.js

```diff
@@ -212,8 +214,8 @@ export class LocalSubscriptionPlatform {
   }
 
   /** @override */
-  supportsCurrentViewer() {
-    return false;
+  getSupportedScoreFactor(unusedFactor) {
+    return 0;
```

Comment:
Promise? We could do a return type `@return {!Promise<number>|number}` and then the using code would do:

```
Promise.resolve(service.getSupportedScoreFactor(...)).then(...)
```

Reply:
I'd rather just fix it to return `Promise.resolve(0);`

Reply:
moot since we went back to async